frontend/src/pages/dashboard/Dashboard.jsx

```diff
@@ -77,21 +77,17 @@ const Dashboard = () => {
           <AdvanceFilter provinceValues={false} schoolTypeValues={false} />
         </Col>
         <Col span={24} align="center">
-          {!pageLoading && chartList ? (
```

Comment:
Hi @navins94 , 
Can we use Loading animation instead of using no data? I want to see the chart template with the loading state before the data is loaded.

Reply:
And also for map data, please add a loading state before the data is loaded. Take a look at wai maps.

Reply:
![Screenshot from 2023-05-02 15-16-34](https://user-images.githubusercontent.com/49269977/235605028-7ec40efb-8630-40ca-93fb-8406d507d0af.png)

And also for the Data View, 
We have year value when showing the history data, please add the year into the data view.